### PR TITLE
Add lstm sigmoid dropout test for dependency optimization

### PR DESCRIPTION
**Pull request by hwangdeyu**
Related to https://github.com/onnx/tensorflow-onnx/issues/2002.
Signed-off-by: Deyu Huang <deyhuang@microsoft.com>

### CODE REVIEW

File: tests/common.py [hunk 1]
```diff
@@ -454,7 +454,7 @@ def group_nodes_by_type(graph):
 
 
 def check_op_count(graph, op_type, expected_count, disabled=True):
-    # FIXME: after switching to grappler some of the op counts are off. Fix later.
+    # The grappler optimization may effect some of the op counts.
```
Comment (fatcat-z) [inline tests/common.py:457]: may effect -> will change

File: tests/test_lstm.py [hunk 2]
```diff
@@ -51,6 +51,7 @@ def new_graph_validator(g):
                 # Skip checks for tflite graphs (no ":" in outputs)
                 return good
             good = good and check_op_count(g, "LSTM", require_lstm_count, disabled=False)
+            # if LSTM op rewriter matchs failed, Loop op will be shown generally.
```
Comment (fatcat-z) [inline tests/test_lstm.py:54]: Let's change it to:

If LSTM op rewriter failed to work, Loop op will be shown in general.

File: tests/common.py (hunk 1 above)
Comment (hwangdeyu) [reply tests/common.py]: Somethings, it will change. So I change it to `may`.

**Review by fatcat-z — APPROVED** [review]
LGTM, thanks!

**Pull request merged**